packages/system/src/styles/transforms.ts

```diff
@@ -14,7 +14,7 @@ export const getTransform = themeGetter<ThemeTransform>({
 })
 
 export interface TransformProps<T extends ITheme = Theme> {
-  transform?: SystemProp<boolean, T>
+  transform?: SystemProp<string, T>
```

Comment:
Fixes:

```tsx
<x.div transform="translateX(20px)">...</x.div>
```

Reply:
The correct type is `true | string` here.